app/views/govuk_publishing_components/components/_step_by_step_nav_related.html.erb

```diff
@@ -1,14 +1,15 @@
 <%
   links ||= []
   pretitle ||= t("govuk_component.step_by_step_nav_related.part_of", default: "Part of")
+  always_display_as_list ||= false
 %>
 <% if links.any? %>
   <div 
     class="gem-c-step-nav-related <%= "gem-c-step-nav-related--singular" if links.length == 1 %>" 
     data-module="track-click">
     <h2 class="gem-c-step-nav-related__heading">
       <span class="gem-c-step-nav-related__pretitle"><%= pretitle %></span>
-      <% if links.length == 1 %>
+      <% if links.length == 1 && always_display_as_list != true %>
```

Comment:
You can write this as `! always_display_as_list`

Reply:
updated